unit/models/transaction.py

```diff
@@ -229,7 +229,7 @@ def from_json_api(_id, _type, attributes, relationships):
         return WireTransactionDTO(_id, date_utils.to_datetime(attributes["createdAt"]), attributes["direction"],
                                   attributes["amount"], attributes["balance"], attributes["summary"],
                                   Counterparty.from_json_api(attributes["counterparty"]), attributes["description"],
-                                  attributes["senderReference"], attributes["referenceForBeneficiary"],
+                                  attributes.get("senderReference"), attributes.get("referenceForBeneficiary"),
```

Comment:
attributes.get("originatorToBeneficiaryInformation"),
                                  attributes.get("senderReference"), attributes.get("referenceForBeneficiary"),
                                  attributes.get("beneficiaryInformation"),
                                  attributes.get("beneficiaryAdviceInformation"),

Reply:
**can you please add the other attributes?**

originator_to_beneficiary_information: str,
                 sender_reference: str, reference_for_beneficiary: str, beneficiary_information: str,
                 beneficiary_advice_information: str,

self.attributes["senderReference"] = sender_reference
        self.attributes["referenceForBeneficiary"] = reference_for_beneficiary
        self.attributes["originatorToBeneficiaryInformation"] = originator_to_beneficiary_information
        self.attributes["beneficiaryInformation"] = beneficiary_information
        self.attributes["beneficiaryAdviceInformation"] = beneficiary_advice_information

Reply:
Added them in the order I found in the docs :)